### ISSUE TITLE
Raspberry Pi tip: Use 2.4 GHz Wi-Fi and avoid USB 3

### ISSUE THREAD

**Issue by benjaoming** [learningequality/kolibri-docs#138]
Users saying to avoid connecting USB 3 devices because of Wi-Fi interference:

> if you're not actually using it, there won't be any interference generated, so, er, don't. 

https://www.raspberrypi.org/forums/viewtopic.php?f=28&t=245607
https://www.raspberrypi.org/forums/viewtopic.php?f=63&t=254615

**Comment by jredrejo** [comment]
This is not a Raspberry Pi problem, it's an universal problem in high-current interfaces if low quality devices are connected. It happens in usb 3.0, usb c, thunderbolt in a $2700 MacBook Pro, any plug at home with a mixer or a kitchen robot working.... 
The solution is not stop using USB 3.0, but using normal or high quality devices on it.

https://www.intel.es/content/www/es/es/products/docs/io/universal-serial-bus/usb3-frequency-interference-paper.html for the exact details.

**Comment by benjaoming** [comment]
Yup, agreed, I know that it's not a Raspberry Pi problem, have pointed that out in another response. The idea was to add the tip in the Raspberry Pi because the interference was raised as an issue by a Raspberry Pi user -- and because we have another issue for adding a Raspberry Pi tip :)
